create-argo/values.yaml

```diff
@@ -1,419 +1,271 @@
-images:
-  pullPolicy: IfNotPresent
+global:
+
+rbac:
+  singleNamespace: true
 createAggregateRoles: false
-crds:
-  install: true
-  keep: true
-singleNamespace: true
-workflow:
-  namespace:
-  serviceAccount:
-    create: true
-    name: ${ARGO_SERVICE_ACCOUNT}
-  rbac:
-    create: true
-executor:
-  image:
-    registry: quay.io
-    repository: argoproj/argoexec
-    tag: ""
-    pullPolicy: ""
-  resources: {}
-  args: []
-  env:
-  - name: RESOURCE_STATE_CHECK_INTERVAL
-    value: 1s
-  - name: WAIT_CONTAINER_STATUS_CHECK_INTERVAL
-    value: 1s
-  securityContext: {}
-%{ if USE_MINIO_STORAGE }
```

Comment:
We lost the s3/artifactRepository config for v3.x compatibility. Unless we are prepared to drop support for old version this is an issue. I'm not 100% sure how the new chart config works but this probably needs to be moved to the `controller.config` entry.

Reply:
Updated too